src/sentry/snuba/metrics/query_builder.py

```diff
@@ -441,18 +417,27 @@ def _build_orderby(self) -> Optional[List[OrderBy]]:
     def __build_totals_and_series_queries(
         self, entity, select, where, groupby, orderby, limit, offset, rollup, intervals_len
     ):
-        # In a series query, we also need to factor in the len of the intervals
-        # array. The number of totals should never get so large that the
-        # intervals exceed MAX_POINTS, however at least a single group.
-        max_totals = max(MAX_POINTS // intervals_len, 1)
-        totals_limit = max_totals
-        if limit and limit.limit < totals_limit:
+        series_limit = None
+        if self._query_definition.include_series:
```

Comment:
Would it simplify the code if we just passed `intervals_len=1` in case we don't want series?

Reply:
How is that the same thing? You could theoretically have series with just a length of 1

Reply:
How would it simplify? `intervals_len` is not used unless we go into the series branch

Reply:
Yeah I mean only keep the series branch (remove the condition), and setting `intervals_len` to 1 where this function is called. I admit it does not make much of a difference though.